server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/UploadStepdefs.java

```diff
@@ -54,6 +55,19 @@ public void userUploadContent(String username) throws Throwable {
             .post("/upload");
     }
 
+    @When("^\"([^\"]*)\" upload a too big content$")
+    public void userUploadTooBigContent(String username) throws Throwable {
+        AccessToken accessToken = userStepdefs.tokenByUser.get(username);
+        RequestSpecification with = with();
+        if (accessToken != null) {
+            with.header("Authorization", accessToken.serialize());
+        }
+        response = with
+            .contentType(ContentType.BINARY)
+            .content(ClassLoader.getSystemResourceAsStream("10Mfile.bin"))
```

Comment:
that's only 20% of the total project size :)
ok... I'm really looking for a better solution. My ConstantInputStream seemed to be a good idea, but I was not able to make it work with restassured or jetty.